test/test_l1_l2_mock_messaging.py

```diff
@@ -103,7 +103,7 @@ def test_send_message_to_l2_execute_without_data():
         }
     )
 
-    assert response.status_code == 400
+    assert response.status_code == 500
```

Comment:
gateway returns 500?

Reply:
Is that a problem? Let's discuss this on the call.